pkg/kubelet/dockertools/kube_docker_client.go

```diff
@@ -110,11 +100,7 @@ func convertEnv(src interface{}) (*docker.Env, error) {
 	return env, nil
 }
 
-func (k *KubeDockerClient) ListContainers(options docker.ListContainersOptions) (apiContainers []docker.APIContainers, err error) {
-	const operation = "list_containers"
-	defer recordOperation(operation, time.Now())
-	defer func() { recordError(operation, err) }()
-
+func (k *kubeDockerClient) ListContainers(options docker.ListContainersOptions) (apiContainers []docker.APIContainers, err error) {
```

Comment:
nit: no need to declare `err` here. It's more clear to return `err` in below. The same applies to other funcions.